### PR TITLE
scispark-200 created OpenDapUtils to read credentials from file

### PR DESCRIPTION
**Pull request by kwhitehall**
- [x] added file
- [x] updates app to access it
- [x] updated app to elegantly exit if bad credentials are given

### CODE REVIEW

File: src/main/scala/org/dia/utils/OpenDapUtils.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        Source.fromFile(cFile)
+          .getLines()
+          .map(p => {
+          val split = p.split("\\s+")
+          (split(0), split(1))
+          }).toList
+      username = cList(0)._2
+      password = cList(1)._2
+    }
+    catch {
```
Comment (rahulpalamuttam) [inline src/main/scala/org/dia/utils/OpenDapUtils.scala]: I'm not a fan of try catches used this way to hide errors. In fact you catch it and just call system.exit anyway. I would recommend to just throw the exception.

Comment (rahulpalamuttam) [inline src/main/scala/org/dia/utils/OpenDapUtils.scala]: Actually ignore the comment, for the purposes of the accumulation app, you have an info string passed to the logger saying the app is ending.

**Comment by kwhitehall** [comment]
Hey @rahulpalamuttam I saw review comments in the email, but nothing here. Are you still reviewing?

**Comment by rahulpalamuttam** [comment]
No im not reviewing anymore 
You can ignore the comment.
+1 from me

**Comment by kwhitehall** [comment]
Ack'ed. Thanks!

**Pull request merged**